connector-j-5/pom.xml

```diff
@@ -29,7 +29,7 @@
     <dependency>
       <groupId>com.google.cloud.sql</groupId>
       <artifactId>jdbc-socket-factory-core</artifactId>
-      <version>1.0.13-SNAPSHOT</version>
+      <version>1.0.13</version>
```

Comment:
Can this use `${project.version}`? (You can consider this for the future instead of now.)

Same for everywhere except the references to the parent.